packages/dx-react-scheduler/src/plugins/month-view.jsx

```diff
@@ -79,6 +79,9 @@ export class MonthView extends React.PureComponent {
       <Plugin
         name="MonthView"
       >
+        <Getter name="currentView" value="month" />
+        <Getter name="intervalCount" value={intervalCount} />
+        <Getter name="firstDayOfWeek" value={firstDayOfWeek} />
```

Comment:
Question: if we use here only one getter `currentViewMeta` with an interface `{ type: Month, intervalCount: 2, firstDayOfWeek: 0 }`.
Is this way bad?

Reply:
I prefer following the [single responsibility principle](https://en.wikipedia.org/wiki/Single_responsibility_principle).